imagejob/server_test.go

```diff
@@ -53,7 +54,7 @@ var fetchCases = []struct {
 
 func setupModule() {
 
-	os.Setenv("GODINARY_FS_BASE", "/tmp/.godinary/")
+	viper.Set("fs_base", "/tmp/.godinary/")
```

Comment:
just curious: why the original var is in uppercase and the new one in lowercase?

Reply:
env vars are uppercase by default, once you pass throgh viper lib you should work lowercase

viper are binded to comandline lowercase like --ssl_dir=/path and binded to env GODINARY_SSL_DIR=/path